### DISCUSSION TITLE
Refseq Annotations in Seqr

### DISCUSSION THREAD

**Discussion by Tulasi08** [broadinstitute/seqr#3916] [General]
Hi Team,

Is it possible to get Refseq transcripts in Seqr? I tried downloading the Cache files and fastq reference files specific for refeseq separately into the docker image's mounted directory. Is there any way to use Refseq annotations instead of Ensembl?

![image](https://github.com/broadinstitute/seqr/assets/122073339/7053def4-5013-4c55-a378-f4d7c56b9821)

Thanks and Regards,
Tulasi S

**Comment by hanars** [comment]
There is no way to get refseq transcripts run in the seqr loading pipeline. However, If you run `./manage.py update_refseq` then when you click into the detailed transcript view for a variant you will see the corresponding refseq transcript(s)
